lib/dal/src/workspace_snapshot/graph.rs

```diff
@@ -1744,7 +1746,7 @@ impl WorkspaceSnapshotGraph {
     pub fn perform_updates(
         &mut self,
         to_rebase_change_set: &ChangeSetPointer,
-        onto: &mut WorkspaceSnapshotGraph,
+        onto: &WorkspaceSnapshotGraph,
```

Comment:
No longer needs to be mutable. This was leftover from when it was in the rebaser. In there, the snapshot needed to be mutable to access a working copy.